lib/matplotlib/axis.py

```diff
@@ -211,7 +211,7 @@ def _apply_tickdir(self, tickdir):
 
     @_api.deprecated("3.5", alternative="axis.set_tick_params")
     def apply_tickdir(self, tickdir):
-        self._apply_tickdir()
+        self._apply_tickdir(tickdir)
```

Comment:
Can we add a test here so this doesn't break again?

Reply:
Yes, added.